src/components/TaskManager.tsx

```diff
@@ -1,14 +1,16 @@
 import TaskForm from "./TaskForm";
 import TaskList from "./TaskList";
-
+import NavigationMenu from "./Navigationmenu";
 function TaskManager(): JSX.Element {
   return (
-    <main>
-      <div className="container-wrapper main-content">
-        <TaskForm />
-        <TaskList />
-      </div>
-    </main>
+    <>
+      <main>
```

Comment:
This `<>` is unnecessary as everything is wrapped in `<main>`. You can also move this `<main>`tag to App.tsx so you can avoid writing it on every page and remove it from TaskList.tsx too.